src/Form.php

```diff
@@ -1754,16 +1817,26 @@ protected function validateField(\SimpleXMLElement $element, $group = null, $val
 			// If the field is required and the value is empty return an error message.
 			if (($value === '') || ($value === null))
 			{
+				$translate = $element['translate_default'] && ((string) $translate == 'true' || (string) $translate == '1');
+
 				if ($element['label'])
 				{
-					$message = Text::_($element['label']);
+					$message = $translate ? $this->getText()->translate($element['label']) : $element['label'];
 				}
 				else
 				{
-					$message = Text::_($element['name']);
+					$message = $translate ? $this->getText()->translate($element['name']) : $element['name'];
 				}
 
-				$message = Text::sprintf('JLIB_FORM_VALIDATE_FIELD_REQUIRED', $message);
+				// TODO - Language strings for our packages should be defined and loaded into the language object
```

Comment:
Hence the reason it's a TODO item.  We'll need to define a standard for where the keys are stored in packages, prefixing standards, and things like that.